src/NLog/Layouts/JSON/JsonLayout.cs

```diff
@@ -333,7 +333,7 @@ private void BeginJsonProperty(StringBuilder sb, string propName, bool beginJson
             }
 
             sb.Append('"');
-            sb.Append(propName);
+            sb.Append(propName.Replace(@"""",@"\"""));
```

Comment:
Could you do this instead: `Targets.DefaultJsonSerializer.AppendStringEscape(sb, propName, false, false);`

Reply:
Done